lib/protocol/events.js

```diff
@@ -203,12 +203,13 @@ function eventsProtocolFactory (
     };
 
 
-    EventsProtocol.prototype.publishGraphStarted = function (graphid, status, nodeId) {
+    EventsProtocol.prototype.publishGraphStarted = function (graphid, data, nodeId) {
         var self = this;
         assert.uuid(graphid);
-        assert.string(status);
-
-        var data = { status: status };
+        assert.object(data);
+        assert.uuid(data.graphId);
```

Comment:
I'm not sure what's the correct usage, I think it's  displayed information, so I think your purpose is not to protect this function, because not checking wouldn't affect this function, if you just want to make sure information display correctly, add the type check in calling method or callback method